internal/provider/aws/health.go

```diff
@@ -39,27 +43,95 @@ func NewRoute53HealthCheckReconciler(client route53iface.Route53API) *Route53Hea
 	}
 }
 
-func (r *Route53HealthCheckReconciler) Reconcile(ctx context.Context, spec provider.HealthCheckSpec, endpoint *externaldns.Endpoint, probeStatus *v1alpha1.HealthCheckStatusProbe, address string) (provider.HealthCheckResult, error) {
+func getTransitionTime(probeConditions []metav1.Condition, conditionType string, conditionStatus metav1.ConditionStatus) metav1.Time {
+	for _, c := range probeConditions {
+		if c.Type == conditionType && c.Status == conditionStatus {
+			return c.LastTransitionTime
+		}
+	}
+	return metav1.Now()
+}
+
+func (r *Route53HealthCheckReconciler) Reconcile(ctx context.Context, spec provider.HealthCheckSpec, endpoint *externaldns.Endpoint, probeStatus *v1alpha1.HealthCheckStatusProbe, address string) provider.HealthCheckResult {
 	healthCheck, exists, err := r.findHealthCheck(ctx, probeStatus)
 	if err != nil {
-		return provider.HealthCheckResult{}, err
+		lastTransition := metav1.Now()
+		if probeStatus != nil {
+			lastTransition = getTransitionTime(probeStatus.Conditions, "ProbeSynced", "False")
+		}
```

Comment:
Is setting this time to now all the time going to be an issue if we get an error? This will end up in the DNSrecord status right?

Reply:
it's only set to now if their is no existing status for that probe, or if the condition has changed.